_sass/blocks/_revenues_subpage_involved.scss

```diff
@@ -0,0 +1,51 @@
+// Revenues Subpage Involved Section
+//
+// This block is where we post videos and social
+// media links on how to become involved
+//
+// Styleguide blocks.revenues_subpage_involved
+
+.revenues_subpage-involved {
```

Comment:
Selector `revenues_subpage-involved` should be written in lowercase with hyphens